meshnets/utils/model_loading.py

```diff
@@ -0,0 +1,51 @@
+"""Methods for loading models"""
+
+import os
+import tempfile
+from typing import Union
+
+import mlflow
+# pylint: disable=unused-import
```

Comment:
Why disable this warning instead of removing the import

Reply:
The import is used implicitly by the `eval` method to evaluate the parameters `tensor([...])